go

```diff
@@ -65,7 +65,7 @@ def update_gems
 end
 
 def update_data
-  exec_cmd 'cd _data && ./import-public.rb'
+  exec_cmd 'cd _data && ruby import-public.rb'
```

Comment:
This could just be `_data/import-public.rb`. The explicit ruby interpreter is only necessary in the staging/prod environment.